cmd/crc/cmd/oc_env.go

```diff
@@ -27,12 +27,6 @@ func runOcEnv(args []string) error {
 		return fmt.Errorf("Error running the oc-env command: %s", err.Error())
 	}
 
-	client := newMachine()
-	consoleResult, err := client.GetConsoleURL()
-	if err != nil {
-		return err
-	}
-	proxyConfig := consoleResult.ClusterConfig.ProxyConfig
```

Comment:
Is `NoProxy` correct without this? `NoProxy` needs to contain `api.xxx`, in order to know what `xxx` is, we need the bundle which is being used, in order to know which bundle is currently being used, we need to look at the machine metadata.

Reply:
Yes this is a mistake, thanks! 
Should I drop this? Hardcode the domain? It is sad to have crc oc-env failing this often only for proxy setup (which are rare setup). 
Should this check only performed when a proxy is enabled ?

Reply:
I dropped the commit. Too complicated for little result.

Reply:
> Hardcode the domain? It is sad to have crc oc-env failing this often only for proxy setup (which are rare setup).
Should this check only performed when a proxy is enabled ?

A mix of these suggestions could work? If proxy is configured, try to get proxy info through machine metadata, if this fails, fallback to hardcoded domain, and warn that NO_PROXY could be incorrect?

Reply:
Will do that in a follow up